test/SDK/MonitoringApplianceEmulatorTests/ViewModels/ChartPropertyControlViewModelTests.cs

```diff
@@ -42,39 +43,75 @@ public class ChartPropertyControlViewModelTests
             new ChartPoint(5.0, 10.0)
         };
 
+        private static readonly Func<double, string> PercentageYFormatter = value => $"{value.ToString(CultureInfo.InvariantCulture)}%";
+
+        private static readonly Func<double, string> NumberFormatter = value => value.ToString(CultureInfo.InvariantCulture);
+
+        private static Func<double, string> dateXFormatter;
+
         public delegate void ChartPointsAssertionDelegate(List<ChartPoint> expectedPoints, List<LiveCharts.ChartPoint> actualPoints);
 
         [TestMethod]
-        public void WhenCreatingNewViewModelForLineChartWithDateTimeXAxisTypeThenItWasInitializedCorrectly()
+        public void WhenCreatingNewViewModelForLineChartWithDateTimeXAxisTypeAndWithNumericYAxisTypeThenItWasInitializedCorrectly()
+        {
+            AssertChartViewModel<LineSeries>(ChartType.LineChart, ChartAxisType.Date, ChartAxisType.Number, DateTimeChartPoints, AssertDateTimeChartPoints);
```

Comment:
>AssertChartViewModel [](start = 12, length = 20)

suggestion to make the tests more readable - 
Use named parameters. so it will be clear which type for X axis and which for Y.  And then split each parameter to a separate line